modules/admin/__init__.py

```diff
@@ -223,19 +223,15 @@ def start(self):
 
         return
 
-
     def stop(self):
         """
-        Stop the admin module
 
-        Cleanup code of the admin module
         """
 
-        self.logger.info("Shutting down".format(self._shortname))
+        self.logger.info("shutting down".format(self._shortname))
```

Comment:
Shutting down würde ich immer (da am Anfang des Logeintrages/Satzes) mit einem großen S schreiben.

Reply:
Ja, und `format()` koennte man auch gleich entfernen oder den `_shortname` noch in die Ausgabe packen, falls das interessant ist.

Reply:
Okay. Habe format() nicht durchgängig entfernt, nur, wenn es sowieso angefasst wurde.